src/components/Pagination/Pagination.js

```diff
@@ -64,8 +66,23 @@ const Pagination = ({
   nextLabel,
   previousLabel,
   footer,
+  tracking,
   ...props
 }) => {
+  const { label, component = 'pagination', customParameters } = tracking || {};
+  const dispatch = useClickTrigger();
+  const handleChange =
+    onChange && label
+      ? clickedPage => {
+          dispatch({
+            label,
+            component: `${component}-${clickedPage}`,
```

Comment:
Is this generally a good idea to have a such default setting? I did this because otherwise there's no way to tell the difference between clicking different pages.

Reply:
I don't think this is ideal. This would make it impossible to filter the event data by the "pagination" component since each component value would be different. It would be   better to track the page number inside the `customParameters`, though I'd consult with the onsite tracking team how they'd prefer to receive this data.

Reply:
Just talked with Ben. Since the consumer also can't customize the label of each page, he would put it directly in the label field. So the dispatch payload would be `{ label: ${pageNumber}, component: 'pagination', customParameters }`